Streams/MemoryStreamWriter.h

```diff
@@ -24,8 +25,8 @@ class MemoryStreamWriter : public SeekableStreamWriter
 	char* streamBuffer;
```

Comment:
Similarly, this can be declared as:
```
char* const streamBuffer;
```
The data is still writable, but the pointer itself is then `const`.